lib/menu/widgets/main_menu_background.dart

```diff
@@ -53,6 +53,12 @@ class _MainMenuBackgroundState extends State<MainMenuBackground>
       });
     ;
 
+    _playerController.addListener(() {
+      if (!_playerController.value.isPlaying) {
+        _playerController.play();
+      }
+    });
```

Comment:
I see that without this workaround we could not made the Earth animation to play. Would be really nice at least to remove Listener on widget dispose.